azure/services/managedclusters/managedclusters.go

```diff
@@ -147,3 +160,75 @@ func (s *Service) Delete(ctx context.Context) error {
 func (s *Service) IsManaged(ctx context.Context) (bool, error) {
 	return true, nil
 }
+
+// ReconcileKubeconfig will reconcile admin kubeconfig and user kubeconfig.
+/*
+  Returns the admin kubeconfig and user kubeconfig
+  If aad is enabled a user kubeconfig will also get generated and stored in the secret <cluster-name>-kubeconfig-user
+  If we disable local accounts for aad clusters we do not have access to admin kubeconfig, hence we need to create
+  the admin kubeconfig by authenticating with the user credentials and retrieving the token for kubeconfig.
+  The token is used to create the admin kubeconfig.
```

Comment:
Is this comment correct or should it be referring to the user kubeconfig instead of the admin one?

Reply:
This comment is correct, if we refer to the user kubeconfig we will need kubelogin to access the cluster.